openedx/features/course_bookmarks/static/course_bookmarks/templates/bookmarks-list.underscore

```diff
@@ -0,0 +1,54 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+
+<% if (bookmarksCollection.length) { %>
+
+    <div class="paging-header"></div>
+
+    <div class='bookmarks-results-list'>
+        <% bookmarksCollection.each(function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item"
+                href="<%- bookmark.blockUrl() %>"
+                aria-labelledby="bookmark-link-<%- index %>"
+                data-bookmark-id="<%- bookmark.get('id') %>"
+                data-component-type="<%- bookmark.get('block_type') %>"
+                data-usage-id="<%- bookmark.get('usage_id') %>"
+                aria-describedby="bookmark-type-<%- index %> bookmark-date-<%- index %>">
+                <div class="list-item-content">
+                    <div class="list-item-left-section">
+                        <h3 id="bookmark-link-<%- index %>" class="list-item-breadcrumbtrail">
+                            <%=
+                                HtmlUtils.HTML(_.map(_.pluck(bookmark.get('path'), 'display_name'), _.escape)
+                                    .concat([_.escape(bookmark.get('display_name'))])
+                                    .join(' <span class="icon fa fa-caret-right" aria-hidden="true"></span><span class="sr">-</span> '))
+                            %>
+                        </h3>
+                        <p id="bookmark-date-<%- index %>" class="list-item-date"> <%- gettext("Bookmarked on") %> <%- humanFriendlyDate(bookmark.get('created')) %> </p>
+                    </div>
+
+                    <p id="bookmark-type-<%- index %>" class="list-item-right-section">
+                        <span aria-hidden="true"><%- gettext("View") %></span>
+                        <span class="icon fa fa-arrow-right" aria-hidden="true"></span>
+                    </p>
+                </div>
+            </a>
+        <% }); %>
+    </div>
+
+    <div class="paging-footer"></div>
+
+<% } else {%>
+
+    <div class="bookmarks-empty">
+        <h3 class="hd-4 bookmarks-empty-header">
+            <span class="icon fa fa-bookmark-o bookmarks-empty-header-icon" aria-hidden="true"></span>
+            <%- gettext("You have not bookmarked any courseware pages yet") %>
+            <br>
+        </h3>
+        <div  class="bookmarks-empty-detail">
+            <span class="bookmarks-empty-detail-title">
+                <%- gettext('Use bookmarks to help you easily return to courseware pages. To bookmark a page, click on "Bookmark this page" beneath the unit title.') %>
```

Comment:
@catong this string was all sorts of wrong before, so I made an attempt to clean it up. Here's how it read before

> Use bookmarks to help you easily return to courseware pages. To bookmark a page, select Bookmark in the upper right corner of that page. To see a list of all your bookmarks, select Bookmarks in the upper left corner of any courseware page.

Let me know if you have recommendations for improving this further.